api/rating/config/routes.json

```diff
@@ -0,0 +1,52 @@
+{
+  "routes": [
```

Comment:
can you delete all the routes we're not using from here .. we only need the create one now